test/logger.go

```diff
@@ -0,0 +1,115 @@
+//go:build e2e
+// +build e2e
+
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+	"path"
+	"path/filepath"
+
+	. "github.com/onsi/gomega"
+
+	"sigs.k8s.io/cluster-api-provider-azure/test/e2e"
+	"sigs.k8s.io/cluster-api/test/framework"
+)
+
+func Fail(message string, callerSkip ...int) {
+	panic(message)
+}
+
+func main() {
+	// needed for ginkgo/gomega which is used by the capi test framework
+	RegisterFailHandler(Fail)
+
+	// required flags
+	kubeconfigPath := flag.String("kube-config", getKubeConfigPath(), "The kubeconfig for the management cluster")
```

Comment:
nit: might be good to match flag names we already have in other tools like clusterctl and capz for consistency, eg:

`--name` for cluster name, `kubeconfig` for kubeconfig, `--resource-group -g` for resource group

Reply:
this makes sense, will do

Reply:
done